src/main/java/frc/team3388/robot/subsystems/ShooterSystem.java

```diff
@@ -0,0 +1,29 @@
+package frc.team3388.robot.subsystems;
+
+
+import com.flash3388.flashlib.io.devices.SpeedController;
+import com.flash3388.flashlib.scheduling.Subsystem;
+
+
+public class ShooterSystem extends Subsystem {
+
+    private final SpeedController motor;
+    private final double MINIMUM_SPEED=0.2;
+    public ShooterSystem(SpeedController motor) {
+        this.motor = motor;
+    }
+
+    public void shoot(double speed) {
```

Comment:
If the speed is `0.0` allow it to be set to the motor. Unlike anything between 0 and 0.2, 0 itself is alright as a speed, because it would just do nothing.